source/renderer/app/config/currencyConfig.coingecko.ts

```diff
@@ -26,6 +26,7 @@ type CurrencyRateGeckoResponse = Array<{
 const id = 'coingecko';
 const name = 'CoinGecko';
 const website = 'https://www.coingecko.com/en/api';
+// Consumes: https://github.com/input-output-hk/wallet-world/blob/d742bd7bd27deca583d6e5a852e7275e35e760b7/nix/cardano-services/deployments.nix#L43
```

Comment:
Hey, this links to a private repo, but Daedalus is open source. Is this link needed in Daedalus? Maybe instead we can link to Daedalus in the private repo?

Reply:
Ah, good point! It was meant as a band aide for comtextual awareness to help track upstream changes.